docs/source/api/user-guide/measurements/canonical-noise-mechanism.ipynb

```diff
@@ -0,0 +1,426 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "from opendp.extras.numpy.canonical import one_sided_uniformly_most_powerful_tests\n",
+    "\n",
+    "ump_tests = one_sided_uniformly_most_powerful_tests(\n",
+    "    theta=theta, d_in=estimate.d_in, d_out=estimate.d_out, size=1_000, alpha=0.05, tail=\"left\"\n",
+    ")\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "plt.plot(list(range(0, 1001)), ump_tests);"
```

Comment:
I'd have a slight preference to replace with with a small table, showing that the values are very close to zero, and then very close to one.

If there were more plots, that would be fine, but with just one, I'd prefer to have a clear migration path to RST. (Which gives us stronger tests, simpler source code etc.)

(A notebook is fine for this PR, I just want to avoid unnecessarily locking ourselves in.)

Reply:
IMO the image is much clearer than a table, and we won't necessarily be locked in to a notebook.